include/seqan3/io/record.hpp

```diff
@@ -36,76 +36,112 @@ namespace seqan3
+    REACT_ERR [[deprecated("Use lower case field names instead.")]],      //!< Please use the field name in lower case.
+    COMMENT [[deprecated("Use lower case field names instead.")]],        //!< Please use the field name in lower case.
+    ALIGNMENT [[deprecated("Use lower case field names instead.")]],      //!< Please use the field name in lower case.
+    REF_ID [[deprecated("Use lower case field names instead.")]],         //!< Please use the field name in lower case.
+    REF_SEQ [[deprecated("Use lower case field names instead.")]],        //!< Please use the field name in lower case.
+    REF_OFFSET [[deprecated("Use lower case field names instead.")]],     //!< Please use the field name in lower case.
+    HEADER_PTR [[deprecated("Use lower case field names instead.")]],     //!< Please use the field name in lower case.
+    FLAG [[deprecated("Use lower case field names instead.")]],           //!< Please use the field name in lower case.
+    MATE [[deprecated("Use lower case field names instead.")]],           //!< Please use the field name in lower case.
+    MAPQ [[deprecated("Use lower case field names instead.")]],           //!< Please use the field name in lower case.
+    CIGAR [[deprecated("Use lower case field names instead.")]],          //!< Please use the field name in lower case.
+    TAGS [[deprecated("Use lower case field names instead.")]],           //!< Please use the field name in lower case.
+    BIT_SCORE [[deprecated("Use lower case field names instead.")]],      //!< Please use the field name in lower case.
+    EVALUE [[deprecated("Use lower case field names instead.")]],         //!< Please use the field name in lower case.
+    USER_DEFINED_0 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_1 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_2 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_3 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_4 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_5 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_6 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_7 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_8 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
+    USER_DEFINED_9 [[deprecated("Use lower case field names instead.")]], //!< Please use the field name in lower case.
```

Comment:
Why not use `SEQAN3_DEPRECATED_310` ?

Reply:
I did not know we have that :see_no_evil: